### PR TITLE
Deprecate saucectl new

### PR DESCRIPTION
**Pull request by alexplischke**
### Description
Remove any mention of `saucectl new`. Since we don't have a true bootstrapping mechanism yet (and `saucectl new` never was), instances of `saucectl new` were replaced by references to the example repos instead.

### CODE REVIEW

File: docs/web-apps/automated-testing/playwright.md [hunk 1]
```diff
@@ -14,143 +14,48 @@ This page provides a walkthrough on how to run [Playwright](https://playwright.d
...
-    ```bash
-    INF Start New Command
-    Choose a framework: Playwright
-    Choose the sauce labs region:
-    ❯ us-west-1
-      eu-central-1
-    ```
-   
-## Run Tests Locally
+You can run a working Cypress sample test in under ten minutes!
```
Comment (FriggaHel) [inline docs/web-apps/automated-testing/playwright.md:19]:
```suggestion
You can run a working Playwright sample test in under ten minutes!
```

File: docs/web-apps/automated-testing/playwright.md [hunk 2]
```diff
@@ -14,143 +14,48 @@ This page provides a walkthrough on how to run [Playwright](https://playwright.d
...
 
-The following steps outline how to run your playwright tests on your local machine with the containerized solution, [Testrunner Toolkit](/testrunner-toolkit):
+### Install `saucectl`
 
-1. Run your playwright tests using `docker` mode by modifying `config.yml` like following:
+Begin by installing the `saucectl` CLI in the root directory of your existing Cypress project or your intended directory.
```
Comment (FriggaHel) [inline docs/web-apps/automated-testing/playwright.md:23]:
```suggestion
Begin by installing the `saucectl` CLI in the root directory of your existing Playwright project or your intended directory.
```

**Review by mhan83 — APPROVED** [review]
Changes look good. Though I wonder, if in the absence of a true bootstrapping option with `saucectl`, whether we want to have explicit instructions for getting `saucectl` working with existing projects. Maybe that belongs in the individual example repos though?

**Review by FriggaHel — APPROVED** [review]

**Review by sweeneyskirt-sl — APPROVED** [review]

**Review by sweeneyskirt-sl — CHANGES_REQUESTED** [review]
Hey @alexplischke -- could you please leave this js-quickstart in for now? Lindsay is edit it to remove references to `saucectl new` and point to the demo repos, but it is linked in the UI, so we don't want to break it just yet.

File: docs/testrunner-toolkit/configuration.md [hunk 3]
```diff
@@ -118,7 +118,7 @@ The `saucectl` command line bundles your root directory (`rootDir` parameter of
 
 ### Excluding Files from the Bundle
 
-The `.sauceignore` file is generated when you run `saucectl new`, allowing you to designate certain files to exclude from bundling.
+The `.sauceignore` allows you to designate certain files to be excluded from bundling.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration.md:121]:
```suggestion
The `.sauceignore` file allows you to designate certain files to be excluded from bundling.
```

**Review by sweeneyskirt-sl — CHANGES_REQUESTED** [review]

**Review by sweeneyskirt-sl — APPROVED** [review]
Made one small wording correction and committed it. Also pulled from master so this branch is in sync. As soon as the CI tests finish, you should be good to go.

**Pull request merged**